doc/tutorial/network_ring.rst

```diff
@@ -117,13 +117,10 @@ and :py:class:`arbor.domain_decomposition`. The first lets Arbor know what hardw
 destribute the work over that hardware. By default, contexts are configured to use 1 thread and domain decompositons to
 divide work equally over all threads.
 
-Step **(12)** initalizes the ``threads`` parameter of :class:`arbor.context` with the ``avail_threads`` flag. By supplying
-this flag, a context is constructed that will use all locally available threads. On your local machine this will match the
-number of logical cores in your system. Especially with large numbers
-of cells you will notice the speed-up. (You could instantiate the recipe with 5000 cells and observe the difference. Don't
-forget to turn of plotting if you do; it will take more time to generate the image then to run the actual simulation!)
-If no domain decomposition is specified, a default one distributing work equally is created. This is sufficient for now.
-You can print the objects to see what defaults they produce on your system.
+Step **(12)** creates a simulation object from the recipe. Optionally, the :py:class:`~arbor.simulation` constructor takes two more
+parameters: a :class:`arbor.context` and a :class:`arbor.domain_decomposition`. In a followup of this tutorial that will be demonstrated.
+For now, it is enough to know that for simulations that don't require customized execution those arguments can be left out. Arbor
+will without any further arguments use all locally available threads.
```

Comment:
```suggestion
Without further arguments Arbor will use all locally available threads.
```
Grammar ;)
also, due to the suggestion quirks, remove the hanging 'Arbor' a line up.